assets/json/level.json

```diff
@@ -0,0 +1,67 @@
+{
+    "level":
+    {
+        "width": 1000,
+        "height": 300,
+        "Lumia": {
+            "posx": 500,
+            "posy": 20,
+            "size": 40
+        },
+        "numplants": 3,
+        "plant_1": {
```

Comment:
can we do an orientation/facing of the plant as well? facing up/down is relevant in the example level

Reply:
there is an example in shipdemo that changes the initial angle, you can take a look at that!